pages/create/[slug]/index.js

```diff
@@ -11,11 +15,27 @@ export default function EmotionEntry({ onAddEmotionDetails, emotionEntries }) {
   const correspondingEntry = emotionEntries.find((entry) => entry.id === id);
   if (!correspondingEntry) return <h2>Page not found!</h2>;
 
+  function handleToggleTooltip() {
+    setIsTooltipOpen(!isTooltipOpen);
+  }
+
   return (
-    <EmotionForm
-      onSubmit={onAddEmotionDetails}
-      correspondingEntry={correspondingEntry}
-      id={id}
-    />
+    <>
+      <Tooltip onToggleTooltip={handleToggleTooltip} show={isTooltipOpen}>
```

Comment:
could you move the state for displaying the tooltip inside the component or do you need to access the state somewhere outside of the component? This would reduce the complexity.